### DISCUSSION TITLE
Thicket v0.4.0 Released - Including Batch Edit

### DISCUSSION THREAD

**Discussion by dvhart** [Thicket-Blender/thicket#50] [General]
Thicket v0.4.0 includes Batch Edit mode, as well as several UI and performance improvements, as well as some long overdue code refactoring and cleanup. Supports Blender 2.91 and Laubwerk 1.36.0. Please give it a shot and let us know what you think.

**Comment by bijoystudio** [comment]
![Blender Preferences 6_2_2021 6_11_48 PM](https://user-images.githubusercontent.com/85232726/120478028-3e401b00-c3ce-11eb-8ce7-ee43879b5711.png)
I cannot find the path .Any video tutorial ?

**Reply by dvhart** [reply]
Hi @bijoystudio , the most likely culprit here is missing the Python Extension from the Laubwerk installation. See the Installation section of the README for the specific steps: https://github.com/Thicket-Blender/thicket/blob/main/README.md

If you've followed those steps and still experience this problem, please open an Issue and provide as much information as you can regarding OS, Blender, Laubwerk, and Thicket versions so we can help you debug it.

**Reply by bijoystudio** [reply]
After custom installing the  Plants Kit Freebie everything load automatically , Thank you.
